lib/ansible/modules/windows/win_domain.py

```diff
@@ -71,6 +71,11 @@
     type: str
     choices: [ Win2003, Win2008, Win2008R2, Win2012, Win2012R2, WinThreshold ]
     version_added: '2.8'
+  log_path:
```

Comment:
The indentations seem off

Reply:
Are you a bot? You're always so fast with reviewing ;-)
Is now fixed.